### PR TITLE
Admin: Organisation members + Admins

### PR DESCRIPTION
**Pull request by atstojanov**
### Describe the background of your pull request

What does your pull request do? Does it solve a bug (which one?), add a feature?

### Additional context

Add any other context

### Governance 

- [ ] Documentation is added
- [ ] Test cases are added or updated

By submitting this pull request I confirm that:
- I've read the contributing document https://github.com/alkem-io/.github/blob/master/CONTRIBUTING.md.
- I've read and understand the Code of Conduct https://github.com/alkem-io/.github/blob/master/CODE_OF_CONDUCT.md.
- I understand that any contributions or suggestions I made may make it into the actual code. I've read the License 
     https://github.com/alkem-io/Coordination/blob/master/LICENSE.
- I understand that submitting the pull request requires agreeing to and signing the contributor license agreement.

### CODE REVIEW

File: src/pages/Admin/Organisation/OrganisationAuthorizationPage.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        entities={{
+          organisationId: organizationId,
+          credential: AuthorizationCredential.OrganisationAdmin,
+        }}
+      >
+        {(entities, actions, state) => (
+          <EditMembers
+            members={entities.allMembers}
+            availableMembers={entities.availableMembers}
+            addingUser={state.addingAdmin}
```
Comment (hero101) [inline src/pages/Admin/Organisation/OrganisationAuthorizationPage.tsx:44]: is `addingAdmin` passed on purpose to `addingUser`? This intent isn't clear.
The same goes for the one below.

File: src/pages/Admin/Organisation/OrganisationCommunityPage.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      >
+        {(entities, actions, state) => (
+          <EditMembers
+            members={entities.allMembers}
+            availableMembers={entities.availableMembers}
+            addingUser={state.addingUser}
+            removingUser={state.removingUser}
+            onAdd={actions.handleAssignMember}
+            onRemove={actions.handleRemoveMember}
+          />
```
Comment (hero101) [inline src/pages/Admin/Organisation/OrganisationCommunityPage.tsx:35]: you probably don't want the user using this to remove himself. In that case the `executor` prop should be used.

File: src/pages/Admin/Organisation/OrganisationAuthorizationPage.tsx [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            removingUser={state.removingAdmin}
+            onAdd={actions.handleAssignAdmin}
+            onRemove={actions.handleRemoveAdmin}
+          />
```
Comment (hero101) [inline src/pages/Admin/Organisation/OrganisationAuthorizationPage.tsx:48]: you probably don't want the user using this to remove himself. In that case the executor prop should be used.

**Review by hero101 — CHANGES_REQUESTED** [review]

File: src/pages/Admin/Organisation/OrganisationAuthorizationPage.tsx (hunk 1 above)
Comment (atstojanov) [reply src/pages/Admin/Organisation/OrganisationAuthorizationPage.tsx:44]: You are right `addingUser/removingUser` is not correct name. I'll rename it to `addingMember/removingMember`.

File: src/pages/Admin/Organisation/OrganisationAuthorizationPage.tsx (hunk 3 above)
Comment (atstojanov) [reply src/pages/Admin/Organisation/OrganisationAuthorizationPage.tsx:48]: I'll enable it. But in my honest opinion user should be able to remove their credentials.

**Review by hero101 — APPROVED** [review]

**Pull request merged**